redis/tests/support/mod.rs

```diff
@@ -45,6 +45,10 @@ enum ServerType {
     Unix,
 }
 
+pub enum RedisModules {
+    RedisJSON,
```

Comment:
The enum should be called `RedisModule` (or just `Module`, @jaymell what do you think?); the variant should be called `RedisJson`.